arm_controller:
  joint_trajectory_action:
    goal_succes: 100
    unit: rad
    joints:
      - arm_joint_1
      - arm_joint_2
      - arm_joint_3
      - arm_joint_4
      - arm_joint_5
    constraints:
      goal_dif: 0.01
      goal_time: 0.6
      arm_joint_1:
        goal: 0.01
      arm_joint_2:
        goal: 0.01
      arm_joint_3:
        goal: 0.01
      arm_joint_4:
        goal: 0.01
      arm_joint_5:
        goal: 0.01

    max_voltage: 4.0
    
    limits:
      arm_joint_1:
        min: 0.0100693
        max: 5.84014
      arm_joint_2:
        min: 0.0100693
        max: 2.61799
      arm_joint_3:
        min: -5.02655
        max: -0.015708
      arm_joint_4:
        min: 0.0221239
        max: 3.4292
      arm_joint_5:
        min: 0.110619
        max: 5.64159